README.md

```diff
@@ -46,16 +46,16 @@ Tracked Image
 Start a terminal shell and install the DeepCell Kiosk wrapper script:
 
 ```bash
-docker run -e DOCKER_TAG=1.1.0 vanvalenlab/kiosk:1.1.0 | sudo bash
+docker run -e DOCKER_TAG=1.1.0 vanvalenlab/kiosk-console:1.1.0 | sudo bash
 ```
 
-To start the kiosk, just run `kiosk` from the terminal shell.
+To start the kiosk, just run `kiosk-console` from the terminal shell.
 
 Check out our [docs](https://deepcell-kiosk.readthedocs.io/en/master/GETTING_STARTED.html) for more information on how to start your own kiosk.
 
 ## Software Architecture
 
-![Kiosk Architecture](https://raw.githubusercontent.com/vanvalenlab/kiosk/master/docs/images/Kiosk_Architecture.png)
+![Kiosk Architecture](https://raw.githubusercontent.com/vanvalenlab/kiosk-console/master/docs/images/Kiosk_Architecture.png)
```

Comment:
I'd like to see if it's possible to add the console into this architecture diagram. I'm not totally sure how to do it, but I think it would help new users contextualize how the pieces fit together.

Reply:
This is a good point. I don't know how to draw in icon for this, but how about:

![Kiosk_Architecture](https://user-images.githubusercontent.com/7930703/77365464-6a183900-6d13-11ea-9290-cc52f3457f2c.png)